### ISSUE TITLE
[FE - Inspect Document] Image of document is completely darked after moving pointer

### ISSUE THREAD

**Issue by nikitakovaliov92** [veraPDF/verapdf-webapp-gui#86]
[Det is Berlin_rPDF_example_2017-02-27.pdf](https://github.com/veraPDF/verapdf-webapp-gui/files/4751760/Det.is.Berlin_rPDF_example_2017-02-27.pdf)

1. Inspect attached PDF file on WCAG 2.1 (All) profile
2. Select image in "inspect-document area"
3. Moving pointer on next image

**Actual result:** Image is darked
![dark](https://user-images.githubusercontent.com/59257413/84140400-3c14dc00-aa5a-11ea-8355-e4783c81ef0b.png)
**Expected result:** Image is not darked

**Comment by nikitakovaliov92** [comment]
Cannot select Figure in "inspect page"
